sjb/actions/script.py

```diff
@@ -30,11 +30,13 @@ class ScriptAction(Action):
     the repository as the working directory.
     """
 
-    def __init__(self, repository, script, title, timeout=7200):
+    def __init__(self, repository, script, title, timeout):
         self.repository = repository
         self.script = script
         if title == None:
             title = _SCRIPT_TITLE
+        if timeout == None:
```

Comment:
So you're fine to use the `timeout=14400` syntax above -- I didn't use it for `_SCRIPT_TITLE` as when you default to a var it ends up leaking out of scope.